main/masterclass.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Masterclass
+---
+
+## Schedule
```

Comment:
No, I wanted to initially, but I don't think it makes sense to put it there, as it might differ a bit per masterclass. I now added an "about", which makes more sense.